backend/schema/corpora_schema.md

```diff
@@ -133,7 +79,7 @@ sex|"male", "female", "mixed", "unknown", or "other"
 ethnicity|string, "na" if non-human, "unknown" if not available
 development\_stage|string, "unknown" if not available
 
-The `tissue` field must be appended with "(cell culture)" or "(organoid)" if appropriate.
+The `tissue` field must be appended with " (cell culture)" or " (organoid)" if appropriate.
```

Comment:
I could use some more information about how a "cell culture" should work. For example, if there are data generated from a culture of H9 cells, what would you expect for _tissue_ & _cell_type_?

Reply:
Thus far, I've been leaving cell_type empty in cell culture. For example, we have an MCF7 dataset where tissue is "UBERON:0001911 (cell culture)" and cell_type is empty.

Reply:
Got it. Would you accept "CL:0000066 (cell culture)" for cell_type of MCF7, or would that be rejected?

Reply:
As written, that would be rejected.

Reply:
I'm not seeing the requirement that would result in the rejected of that. Can you point it out to me, or clarify it in the doc?

Reply:
It says in the table it has to be an UBERON term...though I might not be clear on the relationship between UBERON and CL. To me it looked like CL properly contains UBERON.